### PR TITLE
Story Promo List - Padding update

### PR DESCRIPTION
**Pull request by sareh**
Part of #3055

**Overall change:** Padding fix - update padding-top to not be applied on first li at 1008px+. 

**Code changes:**

- Padding fix - update padding-top to not be applied on first li at 1008px+. 

Fix can be seen in this storybook story:
http://localhost:8181/?path=/story/components-storypromo-storypromolist--default

Screenshot 2020-02-06 at 09 55 32

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-story-promo-list/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 3.0.2 | [PR#x](https://github.com/bbc/psammead/pull/x) Padding fix - update padding-top to not be applied on first li at 1008px+. |
```
Comment (sareh) [inline packages/components/psammead-story-promo-list/CHANGELOG.md]:
```suggestion
| 3.0.2 | [PR#3070](https://github.com/bbc/psammead/pull/3070) Padding fix - update padding-top to not be applied on first li at 1008px+. |
```

**Review by DenisHdz — APPROVED** [review]
LGTM

**Review by j-pendlebury — APPROVED** [review]
Looks great 👍

**Comment by PriyaKR** [comment]
Padding update looks good.

**Pull request merged**